vip_spec.xsd

```diff
@@ -337,8 +340,11 @@ reference elements for polling location
         <xs:element name="Person">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: Types xs:string and InternationalizedText seem
+                   to be used inconsistently here. Suggestions?
```

Comment:
My take on this is that if it's info that someone submitted to file (e.g. their personal name), then that should be a string simply providing what they filled out as is.  In contrast, the "BallotName" for the candidate is the name as it should appear on the ballot, and so can have translations.  Profession is a bit of an exception in that it can appear translated on the ballot (see picture below).  When I first saw that, I was thinking there should also be a "BallotProfession" as distinct from the profession listed in the Person object.

![sample](https://cloud.githubusercontent.com/assets/355822/7279090/3e8a1174-e8ce-11e4-822f-afdd5457cab8.png)